### PR TITLE
Update Pods Form and Pods View placeholder images

### PR DESCRIPTION
**Pull request by nicdford**
Updates the new Pods block placeholder images/content.

### CODE REVIEW

File: src/Pods/Blocks/Types/Form.php [hunk 1]
```diff
@@ -107,7 +107,20 @@ public function render( $attributes = [] ) {
...
 		if ( is_admin() || wp_is_json_request() ) {
-			return __( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' );
+			ob_start();
+			?>
+				<div class="pods-block-placeholder_container">
+					<div class="pods-block-placeholder_content-container">
+						<img src="<?php echo esc_url( PODS_URL ); ?>ui/images/pods-logo-green.svg" alt="Pods logo" class="pods-logo">
+						<div class="pods-block-placeholder_content">
+							<h2 class="pods-block-placeholder_title">Form</h2>
+							<p><?php echo __( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' ); ?></p>
```
Comment (tr1b0t) [inline src/Pods/Blocks/Types/Form.php:117]: * :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '__'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: f451c649dc0aab905937a16c6988bf0c</sub>

File: src/Pods/Blocks/Types/Form.php [hunk 2]
```diff
@@ -107,7 +107,20 @@ public function render( $attributes = [] ) {
...
+						</div>
+					</div>
+					<img src="<?php echo esc_url( PODS_URL ); ?>ui/images/pods-form-placeholder.svg" alt="Generic placeholder image depicting a common form layout" class="pods-block-placeholder_image">
+				</div>
+			<?
```
Comment (tr1b0t) [inline src/Pods/Blocks/Types/Form.php:122]: :no_entry_sign: Short PHP opening tag used; expected "<?php" but found "<?"

<sub>hash: 414f13f801412231af16bdafd2806c3a</sub>

File: src/Pods/Blocks/Types/View.php [hunk 3]
```diff
@@ -126,7 +126,20 @@ public function render( $attributes = [] ) {
...
 		if ( is_admin() || wp_is_json_request() ) {
-			return __( 'No preview is available for this Pods View, you will see it on the frontend.', 'pods' );
+			ob_start();
+			?>
+				<div class="pods-block-placeholder_container">
+					<div class="pods-block-placeholder_content-container">
+						<img src="<?php echo esc_url( PODS_URL ); ?>ui/images/pods-logo-green.svg" alt="Pods logo" class="pods-logo">
+						<div class="pods-block-placeholder_content">
+							<h2 class="pods-block-placeholder_title">View</h2>
+							<p><?php echo __( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' ); ?></p>
```
Comment (tr1b0t) [inline src/Pods/Blocks/Types/View.php:136]: * :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '__'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 1f29064d02bada6d41ad00389af972b4</sub>

File: src/Pods/Blocks/Types/View.php [hunk 4]
```diff
@@ -126,7 +126,20 @@ public function render( $attributes = [] ) {
...
+						</div>
+					</div>
+					<img src="<?php echo esc_url( PODS_URL ); ?>ui/images/pods-view-placeholder.svg" alt="Generic placeholder image depicting a common form layout" class="pods-block-placeholder_image">
+				</div>
+			<?
```
Comment (tr1b0t) [inline src/Pods/Blocks/Types/View.php:141]: :no_entry_sign: Short PHP opening tag used; expected "<?php" but found "<?"

<sub>hash: 7cfef1f1545ef0fa89c7d1b4e79accca</sub>

File: src/Pods/Blocks/Types/Form.php [hunk 5]
```diff
@@ -107,7 +107,20 @@ public function render( $attributes = [] ) {
 
 		// Prevent any previews of this block.
 		if ( is_admin() || wp_is_json_request() ) {
-			return __( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' );
+			ob_start();
+			?>
+				<div class="pods-block-placeholder_container">
+					<div class="pods-block-placeholder_content-container">
+						<img src="<?php echo esc_url( PODS_URL ); ?>ui/images/pods-logo-green.svg" alt="Pods logo" class="pods-logo">
```
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/Form.php:114]:
```suggestion
						<img src="<?php echo esc_url( PODS_URL . 'ui/images/pods-logo-green.svg' ); ?>" alt="<?php esc_attr_e( 'Pods logo', 'pods' ); ?>" class="pods-logo">
```

File: src/Pods/Blocks/Types/Form.php [hunk 6]
```diff
@@ -107,7 +107,20 @@ public function render( $attributes = [] ) {
...
+						<div class="pods-block-placeholder_content">
+							<h2 class="pods-block-placeholder_title">Form</h2>
```
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/Form.php:116]:
```suggestion
							<h2 class="pods-block-placeholder_title"><?php esc_html_e( 'Form', 'pods' ); ?></h2>
```

File: src/Pods/Blocks/Types/Form.php [hunk 7]
```diff
@@ -107,7 +107,20 @@ public function render( $attributes = [] ) {
...
+							<p><?php echo __( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' ); ?></p>
+						</div>
+					</div>
+					<img src="<?php echo esc_url( PODS_URL ); ?>ui/images/pods-form-placeholder.svg" alt="Generic placeholder image depicting a common form layout" class="pods-block-placeholder_image">
```
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/Form.php:120]:
```suggestion
					<img src="<?php echo esc_url( PODS_URL . 'ui/images/pods-form-placeholder.svg' ); ?>" alt="<?php esc_attr_e( 'Generic placeholder image depicting a common form layout', 'pods' ); ?>" class="pods-block-placeholder_image">
```

File: src/Pods/Blocks/Types/View.php [hunk 8]
```diff
@@ -126,7 +126,20 @@ public function render( $attributes = [] ) {
 
 		// Prevent any previews of this block.
 		if ( is_admin() || wp_is_json_request() ) {
-			return __( 'No preview is available for this Pods View, you will see it on the frontend.', 'pods' );
+			ob_start();
+			?>
+				<div class="pods-block-placeholder_container">
+					<div class="pods-block-placeholder_content-container">
+						<img src="<?php echo esc_url( PODS_URL ); ?>ui/images/pods-logo-green.svg" alt="Pods logo" class="pods-logo">
```
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/View.php:133]:
```suggestion
						<img src="<?php echo esc_url( PODS_URL . 'ui/images/pods-logo-green.svg' ); ?>" alt="<?php esc_attr_e( 'Pods logo', 'pods' ); ?>" class="pods-logo">
```

File: src/Pods/Blocks/Types/View.php (hunk 3 above)
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/View.php:136]:
```suggestion
							<p><?php _e( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' ); ?></p>
```

File: src/Pods/Blocks/Types/Form.php (hunk 1 above)
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/Form.php:117]:
```suggestion
							<p><?php esc_html_e( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' ); ?></p>
```

File: src/Pods/Blocks/Types/View.php [hunk 9]
```diff
@@ -126,7 +126,20 @@ public function render( $attributes = [] ) {
...
+						<div class="pods-block-placeholder_content">
+							<h2 class="pods-block-placeholder_title">View</h2>
+							<p><?php echo __( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' ); ?></p>
+						</div>
+					</div>
+					<img src="<?php echo esc_url( PODS_URL ); ?>ui/images/pods-view-placeholder.svg" alt="Generic placeholder image depicting a common form layout" class="pods-block-placeholder_image">
```
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/View.php:139]:
```suggestion
					<img src="<?php echo esc_url( PODS_URL . 'ui/images/pods-view-placeholder.svg' ); ?>" alt="<?php esc_attr_e( 'Generic placeholder image depicting a common view layout', 'pods' ); ?>" class="pods-block-placeholder_image">
```

File: src/Pods/Blocks/Types/Form.php (hunk 2 above)
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/Form.php:122]:
```suggestion
			<?php
```

File: src/Pods/Blocks/Types/View.php (hunk 4 above)
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/View.php:141]:
```suggestion
			<?php
```

File: src/Pods/Blocks/Types/View.php [hunk 10]
```diff
@@ -126,7 +126,20 @@ public function render( $attributes = [] ) {
...
 		// Prevent any previews of this block.
 		if ( is_admin() || wp_is_json_request() ) {
-			return __( 'No preview is available for this Pods View, you will see it on the frontend.', 'pods' );
+			ob_start();
+			?>
+				<div class="pods-block-placeholder_container">
+					<div class="pods-block-placeholder_content-container">
+						<img src="<?php echo esc_url( PODS_URL ); ?>ui/images/pods-logo-green.svg" alt="Pods logo" class="pods-logo">
+						<div class="pods-block-placeholder_content">
+							<h2 class="pods-block-placeholder_title">View</h2>
```
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/View.php:135]:
```suggestion
							<h2 class="pods-block-placeholder_title"><?php esc_html_e( 'View', 'pods' ); ?></h2>
```

File: ui/images/pods-form-placeholder.svg [hunk 11]
```diff
@@ -0,0 +1,46 @@
+<svg width="530" height="253" viewBox="0 0 530 253" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (sc0ttkclark) [inline ui/images/pods-form-placeholder.svg:1]: @nicdford Can we get a faux submit button on the right side of the form image here?

Comment (nicdford) [reply ui/images/pods-form-placeholder.svg]: Absolutely!

File: src/Pods/Blocks/Types/Form.php [hunk 12]
```diff
@@ -107,7 +107,20 @@ public function render( $attributes = [] ) {
...
 		if ( is_admin() || wp_is_json_request() ) {
-			return __( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' );
+			ob_start();
+			?>
+				<div class="pods-block-placeholder_container">
+					<div class="pods-block-placeholder_content-container">
+						<img src="<?php echo esc_url( PODS_URL . 'ui/images/pods-logo-green.svg' ); ?>" alt="<?php esc_attr_e( 'Pods logo', 'pods' ); ?>" class="pods-logo">
+						<div class="pods-block-placeholder_content">
+							<h2 class="pods-block-placeholder_title"><?php esc_html_e( 'Form', 'pods' ); ?></h2>
+							<p><?php esc_html_e( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' ); ?></p>
```
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/Form.php:117]:
```suggestion
							<p><?php esc_html_e( 'No preview is available for this Pods View, you will see it when you view or preview this post on the front of your site.', 'pods' ); ?></p>
```

File: src/Pods/Blocks/Types/View.php [hunk 13]
```diff
@@ -126,7 +126,20 @@ public function render( $attributes = [] ) {
...
 		if ( is_admin() || wp_is_json_request() ) {
-			return __( 'No preview is available for this Pods View, you will see it on the frontend.', 'pods' );
+			ob_start();
+			?>
+				<div class="pods-block-placeholder_container">
+					<div class="pods-block-placeholder_content-container">
+						<img src="<?php echo esc_url( PODS_URL . 'ui/images/pods-logo-green.svg' ); ?>" alt="<?php esc_attr_e( 'Pods logo', 'pods' ); ?>" class="pods-logo">
+						<div class="pods-block-placeholder_content">
+							<h2 class="pods-block-placeholder_title"><?php esc_html_e( 'View', 'pods' ); ?></h2>
+							<p><?php esc_html_e( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' ); ?></p>
```
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/View.php:136]:
```suggestion
							<p><?php esc_html_e( 'No preview is available for this Pods Form, you will see it when you view or preview this post on the front of your site.', 'pods' ); ?></p>
```

File: src/Pods/Blocks/Types/Form.php [hunk 14]
```diff
@@ -107,7 +107,20 @@ public function render( $attributes = [] ) {
...
 		if ( is_admin() || wp_is_json_request() ) {
-			return __( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' );
+			ob_start();
+			?>
+				<div class="pods-block-placeholder_container">
+					<div class="pods-block-placeholder_content-container">
+						<img src="<?php echo esc_url( PODS_URL . 'ui/images/pods-logo-green.svg' ); ?>" alt="<?php esc_attr_e( 'Pods logo', 'pods' ); ?>" class="pods-logo">
+						<div class="pods-block-placeholder_content">
+							<h2 class="pods-block-placeholder_title"><?php esc_html_e( 'Form', 'pods' ); ?></h2>
+							<p><?php esc_html_e( 'No preview is available for this Pods View, you will see it when you view or preview this post on the front of your site.', 'pods' ); ?></p>
```
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/Form.php:117]:
```suggestion
							<p><?php esc_html_e( 'No preview is available for this Pods Form, you will see it when you view or preview this post on the front of your site.', 'pods' ); ?></p>
```

File: src/Pods/Blocks/Types/View.php [hunk 15]
```diff
@@ -126,7 +126,20 @@ public function render( $attributes = [] ) {
...
 		if ( is_admin() || wp_is_json_request() ) {
-			return __( 'No preview is available for this Pods View, you will see it on the frontend.', 'pods' );
+			ob_start();
+			?>
+				<div class="pods-block-placeholder_container">
+					<div class="pods-block-placeholder_content-container">
+						<img src="<?php echo esc_url( PODS_URL . 'ui/images/pods-logo-green.svg' ); ?>" alt="<?php esc_attr_e( 'Pods logo', 'pods' ); ?>" class="pods-logo">
+						<div class="pods-block-placeholder_content">
+							<h2 class="pods-block-placeholder_title"><?php esc_html_e( 'View', 'pods' ); ?></h2>
+							<p><?php esc_html_e( 'No preview is available for this Pods Form, you will see it when you view or preview this post on the front of your site.', 'pods' ); ?></p>
```
Comment (sc0ttkclark) [inline src/Pods/Blocks/Types/View.php:136]:
```suggestion
							<p><?php esc_html_e( 'No preview is available for this Pods View, you will see it when you view or preview this on the front of your site.', 'pods' ); ?></p>
```

**Pull request merged**